plugins/social_network/social_network.cpp

```diff
@@ -281,6 +321,30 @@ namespace golos { namespace plugins { namespace social_network {
                     con.block_number = db.head_block_num();
                 });
             }
+
+            if (db.has_index<comment_last_update_index>()) {
+                const auto& clu_idx = db.get_index<comment_last_update_index>().indices().get<golos::plugins::social_network::by_comment>();
+                auto clu_itr = clu_idx.find(comment_id_type(comment.id));
+                auto now = db.head_block_time();
+                if (clu_itr != clu_idx.end()) {
+                    // Edit case
+                    db.modify(*clu_itr, [&](comment_last_update_object& clu) {
+                        clu.active = now;
+                        clu.last_update = clu.active;
+                    }); 
+                } else {
+                    // Creation case
+                    db.create<comment_last_update_object>([&](comment_last_update_object& clu) {
+                        clu.comment = comment.id;
+                        clu.author = comment.author;
+                        clu.parent_author = comment.parent_author;
+                        clu.active = now;
+                        clu.last_update = clu.active;
+                    });
```

Comment:
Create and modify like in `activate_parent_comments()`, is't it?
Difference only with `last_update`, but you can fix it with flag parameter in the function-call.

Reply:
@afalaleev It's no problem to move it into a function, but we have very many same places in code. For instance, it is inconvenient to use constructions like `get_index<...>.indices.get.find`, let's define in database the functions like `get_comment_last_update_object(comment_id_type)` for all db types.

Reply:
For common objects there are already exists get_XXX: get_comment, get_account ...

You can add get_comment_last_update_object in plugin scope too.